### PR TITLE
Update graphql to v0.4.13

### PR DESCRIPTION
**Pull request by plievone**
graphql/language/schema was merged with graphql/language in graphql-js v0.4.3.
Also fixed needed `npm run update-schema` in root by adding `babel` as a devDependecy, which provides `node_modules/.bin/babel-node` executable. The changes in `babel-relay-plugin/lib/RelayQLAST.js` are perhaps due to differing babel helper versions. This version is built using a clean install.

### CODE REVIEW

File: scripts/babel-relay-plugin/src/__tests__/testschema.rfc.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
       "mutationType": {
         "name": "Mutation"
       },
+      "subscriptionType": null,
```
Comment (plievone) [inline scripts/babel-relay-plugin/src/__tests__/testschema.rfc.json]: Should I generate the subscription type here (by adding `Subscription` name to a couple of `utilities.buildASTSchema()` calls next to `Mutation`).

**Comment by josephsavona** [comment]
@facebook-github-bot import

**Comment by josephsavona** [comment]
@plievone thanks!!!!

**Comment by plievone** [comment]
Thanks and have nice weekend

**Pull request closed without merging**